manifests/Blended Ships.yaml

```
@@ -0,0 +1,13 @@
+name: Blended Ships
+authors: mOctave
+homepage: https://github.com/mOctave/blended-ships/
+license: AGPL-3+
```

Please use the longer identifiers which aren't deprecated
```suggestion
license: AGPL-3.0-or-later
```